QUEST.tsv

```diff
@@ -2391,48 +2391,48 @@ QUEST_20150803_002398	There is a manager that manages the fortress of the great
+QUEST_20150918_002417	The people of Fedimian live by the teachings of the Maven.{nl}But the Demon Lord dirtied it ..The Revalators sure punished him for the Goddess. {nl}Now, what do you want me to check?{nl}
+QUEST_20150918_002418	I heard that some Revelator saved the Mage Tower. {nl}Fedimian should be safe now, right? {nl}Anyway, you might find some helpful items so look around. {nl}
+QUEST_20150918_002419	The expectation for the Revelator from all the people of Fedimian is rising. {nl}A news spread that a certain Revelator defeated the Demon Lord holding the Cathedral.{nl}Anyway, you don't ncesassrily have to buy something so feel free to look around.
+QUEST_20150918_002420	Seems like something is in that grass.
+QUEST_20150918_002421	What was she looking for even when she was dying? {nl}There's definitely a tragic story behind all this.
+QUEST_20150918_002422	Then something snapped. {n1}A fellow clergyman, I can't remember who, had murdered Dion in his sleep with a mace and I… {nl}Everyone… {nl}No one can remember why they wanted to become priest in the past anymore.
+QUEST_20150918_002423	Each Goddess Statue is known to have different abilities depending on which Goddess it depicts. {nl}For example, Goddess Zemyna gives blessing that enhaces the ability you want.
+QUEST_20150918_002424	But Ausurine Goddess staute of Klaipeda is a bit strange. {nl}The statue moves you to your desired area even though it doesn't depict Goddess Vakarine.
+QUEST_20150918_002425	Go down quickly. It's dangerous. {nl}This place is empty.
+QUEST_20150918_002426	I can't feel any spell.
+QUEST_20150918_002427	It's empty now.
+QUEST_20150918_002428	Underground Chapel Dungeon
+QUEST_20150918_002429	Underground Chapel Dungeon(Recommended LV : 40). Only you and your party members can enter together.{nl}Are you sure you want to enter?
+QUEST_20150918_002430	Fallen Legwyn Family Dungeon
+QUEST_20150918_002431	Fallen Legwyn Family Dungeon(Recommended LV : 132). Only you and your party members can enter together.{nl}Are you sure you want to enter?
+QUEST_20150918_002432	Grand Mage Tower Dungeon
+QUEST_20150918_002433	Grand Mage Tower Dungeon(Recommended LV : 190). Only you and your party members can enter together.{nl}Are you sure you want to enter?
+QUEST_20150918_002434	Catacomb Underground Dungeon
+QUEST_20150918_002435	Catacomb Underground Dungeon(Recommended LV : 200). Only you and your party members can enter together.{nl}Are you sure you want to enter?
+QUEST_20150918_002436	I'm looking over Shop because father is busy. {nl}I many not be as good as my father but my touch is precise. 
+QUEST_20150918_002437	Look around! {nl}It's a bit crowed, but many items are waiting to meet its owner.
+QUEST_20150918_002438	Warehouse Keeper Jadan
+QUEST_20150918_002439	Welcome. It's a bit hustling, right? ? {nl} Please understand as we're trying best effort to restore the place.{nl}Anyway, what do you need me for?
```

Comment:
"Welcome. It sure is busy here, right?{nl} Please be patient with us as we're trying our best to restore the area.{nl}Anyway, what do you need me for?"